quinn/src/connection.rs

```diff
@@ -409,6 +410,14 @@ where
             .peer_identity()
     }
 
+    /// A stable identifier for this peer
```

Comment:
```suggestion
    /// A stable identifier for this connection
```

It's entirely possible to have multiple connections to a single peer.

Reply:
Ah, good point